practice-two/src/components/Information/index.tsx

```diff
@@ -0,0 +1,118 @@
+// react
+import React, { useMemo } from "react";
+
+// types
+import { StudentProfile, IStudentInformation } from "@type/studentProfile";
+
+// assets
+import { email, phone, teacher } from "@assets/icon";
+
+// css
+import "./Information.css";
+
+export interface InformationStudentProps {
+    student: StudentProfile;
+    classmates: IStudentInformation[];
+}
+
+const filterClassmates = (
+    allClassmates: IStudentInformation[],
+    currentStudent: StudentProfile
+): IStudentInformation[] => {
+    return allClassmates.filter(
+        (classmate) =>
+            classmate.classes === currentStudent.classes &&
+            classmate.studentID !== currentStudent.studentID
+    );
+};
+
+const InformationStudent: React.FC<InformationStudentProps> = React.memo(
+    ({ student, classmates }) => {
+        const classmatesList = useMemo(
+            () => filterClassmates(classmates, student),
+            [classmates, student]
+        );
+        const displayCount = 5;
+
+        const renderClassmates = () => {
+            return classmatesList
+                .slice(0, displayCount)
+                .map((classmate) => (
+                    <img
+                        key={classmate.studentID}
+                        src={classmate.imageUrl}
+                        alt={classmate.name}
+                        className="classmate-image"
+                    />
+                ));
+        };
+
+        const renderMoreClassmates = () => {
+            if (classmatesList.length > displayCount) {
+                return (
+                    <span className="more-classmates">
+                        +{classmatesList.length - displayCount} more
+                    </span>
+                );
+            }
+            return null;
+        };
+
+        return (
+            <div className="information-student">
+                <div className="info-student">
+                    <span className="student-id">{student.studentID}</span>
+                    <img
+                        src={student.imageUrl}
+                        alt="student-img"
+                        className="student-image"
+                    />
+                    <div className="name-item">
+                        <span className="student-name">{student.name}</span>
```

Comment:
do you see student name have label and value the same gender or another info. please split component for re-use
![image](https://github.com/ngocphong230801/react-practice/assets/26370013/3e383f29-a4ce-4b76-8cdf-6a9c84f0ce71)